src/core/lv_obj_tree.c

```diff
@@ -191,14 +191,19 @@ void lv_obj_move_to_index(lv_obj_t * obj, int32_t index)
 {
     LV_ASSERT_OBJ(obj, MY_CLASS);
 
+    lv_obj_t * parent = lv_obj_get_parent(obj);
+
+    if(!parent) {
+        LV_LOG_ERROR("parent is NULL");
```

Comment:
```suggestion
        LV_LOG_WARN("parent is NULL");
```

Errors are used when LVGL halts.

Reply:
done